src/main/java/org/fcrepo/camel/FcrepoHeaders.java

```diff
@@ -26,6 +26,8 @@
 
     public static final String FCREPO_TRANSFORM = "FCREPO_TRANSFORM";
 
+    public static final String HTTP_PREFER = "CamelHttpPrefer";
```

Comment:
Sort of. I began using ALL_CAPS because I had seen that elsewhere, but other camel components seem to use the `CamelComponentHeader` convention, especially the HTTP headers. There is a common pattern to remove all HTTP headers with a line like

    removeHeaders("CamelHttp*")

By following that convention, the `CamelHttpPrefer` header fits more easily into existing patterns.

Really, though, the other headers should be changed to:

    CamelFcrepoTransform
    CamelFcrepoBaseUrl
    CamelFcrepoIdentifier

I would recommend changing those headers, too. The main question, though, is whether `CamelHttpPrefer` shouldn't instead by `CamelFcrepoPrefer`. And the more I think about it, the more I think it should be `CamelFcrepoPrefer`.